examples/rabbitmq-consumer-with-qos-settings/rabbitmq-consumer-with-qos-settings.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on consuming messages from a RabbitMQ queue with specific "quality of service" settings.
```

Comment:
Suggested edit: BBE on consuming messages from a RabbitMQ queue with specific quality of service (QoS) settings.